ui/src/app/dashboard/controllers/DashCtrl.js

```diff
@@ -52,18 +52,21 @@
         });
       }
     };
-    
+
     $scope.addDownload = function(keyEvent) {
   if (keyEvent.which === 13)
     $scope.addLink();
   }
     var getActiveDownloads = function() {
       DashService.getDownloads().then(function (response) {
         var data = response.data;
-        $scope.downloads = _.filter(data, function(d) {return d.status==0});
-        $scope.downloads = _.map($scope.downloads, function(element) {
-             return _.extend({}, element, {progress: 0});
-        });
+        $scope.downloads = _.reduce(data, function(queue, d) {
```

Comment:
Agree :+1:

Reply:
It doesn't break any functionality? just test it out once.

Reply:
Tested, works fine! :D